dev-docs/modules/userId.md

```diff
@@ -489,6 +489,43 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### AMX RTB ID
+
+The AMX RTB ID is a first-party identifier designed for publishers using the AMX RTB adapter. For more information please contact [prebid@amxrtb.com](prebid@amxrtb.com)
+
+### AMX RTB ID Configuration
```

Comment:
Can you make this a level 4 heading (####) rather than level 3 so it's nested under AMX RTB ID? Thanks.